src/sidebar/components/annotation-header.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+var annotationMetadata = require('../annotation-metadata');
+var memoize = require('../util/memoize');
+var persona = require('../filter/persona');
+
+// @ngInject
+function AnnotationHeaderController(groups, settings, serviceUrl) {
```

Comment:
So `AnnotationHeaderController` has six dependencies: `annotationMetadata`, `memoize`, `persona`, `groups`, `settings` and `serviceUrl`. I wonder if it might be possible in future to refactor things in order to reduce the number of dependencies here?